wdl2cwl/main.py

```diff
@@ -300,7 +300,10 @@ def get_memory_literal(self, memory_runtime: WDL.Expr.String) -> float:
         if memory_runtime.literal is None:
             _, placeholder, unit, _ = memory_runtime.parts
             with WDLSourceLine(placeholder, ConversionException):
-                value_name = self.get_expr_get(placeholder.expr)  # type: ignore
+                if isinstance(placeholder.expr, WDL.Expr.Get):
+                    value_name = self.get_expr_get(placeholder.expr)  # type: ignore
+                else:
+                    value_name = self.get_expr_apply(placeholder.expr)
```

Comment:
support memory literal of type WDL.Expr.String with a placeholder with expression apply (like in the case of select_first).